src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -167,11 +167,11 @@ const useCloudEnvironmentPolling = namespace => {
   }
   const load = async maybeStale => {
     try {
-      const [newDisks, newRuntimes, galaxyDisks] = await Promise.all([
-        Ajax(signal).Disks.list({ googleProject: namespace, creator: getUser().email }),
-        Ajax(signal).Runtimes.list({ googleProject: namespace, creator: getUser().email }),
-        Ajax(signal).Disks.list({ googleProject: namespace, creator: getUser().email, saturnApplication: 'galaxy' })
-      ])
+      const [newDisks, newRuntimes, galaxyDisks] = googleProject ? await Promise.all([
+        Ajax(signal).Disks.list({ googleProject, creator: getUser().email }),
+        Ajax(signal).Runtimes.list({ googleProject, creator: getUser().email }),
+        Ajax(signal).Disks.list({ googleProject, creator: getUser().email, saturnApplication: 'galaxy' })
+      ]) : [[], [], []]
```

Comment:
Nice choice here. I like how the shape of the absent values so obviously matches the expectation.